### PR TITLE
Add rollbar js

### PR DESCRIPTION
**Pull request by adamrmelnyk**

### CODE REVIEW

File: app/assets/javascripts/rollbar.ready.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var _rollbarConfig = {
+      endpoint: RollbarConfig.js_endpoint,
```
Comment (MathieuGilbert) [inline app/assets/javascripts/rollbar.ready.js]: Could you clean up the formatting in this file? 2-space indents, fields aligned in same column, closing brackets in same column as the start of the key.

**Comment by MathieuGilbert** [comment]
The version number should also get bumped in `version.rb` with any updates to it.

**Comment by MathieuGilbert** [comment]
:poop:

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**